mantidproject/mantid
# Add Zoom System Tests + Reorg SANS System Tests

**Description of work.**

Desc:
- Fixed sys.path in systemtests
- Called setup and teardown for all tests
- Re-organised ISIS_SANS system tests + added a helper which fixes up the path to data without affecting everyone
- Added ZOOM test

**To test:**
- Check CI passes
- Check one of the renamed files still ran on CI

Part of #28987 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
